tests/utimensat/01.t

```diff
@@ -0,0 +1,54 @@
+#! /bin/sh
+# $FreeBSD$
+
+desc="utimensat with UTIME_NOW will set the will set typestamps to now"
+
+dir=`dirname $0`
+. ${dir}/../misc.sh
+echo "1..7"
+
+require utimensat
+
+UTIME_NOW=-1
```

Comment:
It would be better for these constants to be symbolic and referenced in the C code -- there's no telling whether or not all of the constants are defined with the same values.